cmd/sign-executable/sign-executable.go

```diff
@@ -176,7 +177,8 @@ func signExecutable(ctx context.Context, opts signOpts) error {
 
 	signedZipPath := path.Join(tempDir, "evergreen_signed.zip")
 	if err = signWithNotaryClient(ctx, toSignPath, signedZipPath, opts); err != nil {
-		return errors.Wrap(err, "signing with client")
+		fmt.Printf("code signing failed: %s", err.Error())
```

Comment:
Should this print to stderr instead of stdout so it shows up in the logs more obviously when it fails?

Reply:
done